stock-tracker/src/features/latestNews/redux/actions.ts

```diff
@@ -1,15 +1,14 @@
 import { SET_NEWS, SET_LOADING_NEWS, SET_ERROR_NEWS } from "./actionTypes";
-import * as constants from "./actionTypes";
 import { ActionWithPayload, Action } from "../../../utils/actions";
 
-export type LatestNews = {
+export interface LatestNews {
```

Comment:
Is it necessary to change it to `interface`? I think we discussed before, those two do not make a huge difference

Reply:
Just preference.